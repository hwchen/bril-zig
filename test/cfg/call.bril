../../bril/test/interp/core/call.bril