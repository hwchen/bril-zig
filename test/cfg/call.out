{"b000":["b001"]}
